pcap/doc.go

```diff
@@ -12,6 +12,12 @@ This package is meant to be used with its parent,
 http://github.com/google/gopacket, although it can also be used independently
 if you just want to get packet data from the wire.
 
+Depending on libpcap version, os support, or file timestamp resolution,
+nanosecond resolution is used for the internal timestamps. Returned timestamps
+are always scaled to nanosecond resolution due to the usage of time.Time.
+libpcap must be at least version 1.5 to support nanosecond timestamps. OpenLive
+suports only microsecond resolution.
```

Comment:
supports

Reply:
Omg I stared for minutes at this and had to ask my wife who immediately pointed out that there is a p missing o_O

Reply:
Just think, if I hadn't been here to review the code, an insignificant typo may have made it into a comment :P 

This is how I know I'm really useful as a reviewer :D

Reply:
fixed